docs/en/inference/convert_tutorial.md

```diff
@@ -1,17 +1,32 @@
 ## Inference - Model Conversion Tutorial
 
+MindOCR Inference support infer from MindOCR trained models and third-party(PaddelOCR and mmOCR) models.
+
 ### 1. MindOCR models
 
 The inference of MindOCR models supports [MindSpore Lite](https://www.mindspore.cn/lite) backend.
 
 ```mermaid
 graph LR;
-    ckpt --> |export| MindIR --> |"converter_lite(offline conversion)"| o[MindIR];
+    A[MindOCR models] -- export --> B[MindIR] -- converter_lite --> C[MindSpore Lite MindIR];
 ```
 
 #### 1.1 Model Export
 
-Before inference, it is necessary to [export](https://github.com/mindspore-lab/mindocr/blob/main/configs/README.md) the trained ckpt to a MindIR file, which stores the model structure and weight parameters.
+Before inference, it is necessary to export the trained ckpt to a [MindIR](https://www.mindspore.cn/docs/zh-CN/r2.0.0-alpha/design/mindir.html) file. Please run `tools/export.py`.
```

Comment:
change to the English link:
https://www.mindspore.cn/docs/en/r2.0/design/mindir.html